dutch/bos/ontvoering/meehelpen/klus/geld-verdienen.md

```diff
@@ -2,4 +2,8 @@ In eerste instantie geloven de mannen je niet maar als ze je Rolex om je arm
 zien zijn ze snel geïnteresseerd. Wie is deze man?
 
 De mannen kijken elkaar aan en na een korte blik vraagt de langste man:
-Voor wie werk jij dan eigenlijk en heb je nog meer klussen?
\ No newline at end of file
+Voor wie werk jij dan eigenlijk en heb je nog meer klussen?
+
+- Ik ben Lucien de zoon van Carmen van Walraven en ik kom [wraak nemen!](wraak/wraak.md)
+
+- k ben Ruben leider van de grootste mensensmokkerlaarsbende van Nederland en ik heb nog [een mooie klus voor één van jullie.](mensensmokkel/mensensmokkel.md)
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.